CHANGELOG/CHANGELOG-v1.3.0.md

```diff
@@ -0,0 +1,99 @@
+            "max_retries": 1,
+        }
+    ))
+```
+
+### New Deployment Type
+A couple releases ago, we introduced a new Flyte [executable](https://github.com/flyteorg/flyte/blob/master/cmd/main.go) that combined all the functionality of Flyte's backend into one command. This simplifies the deployment in that only one image needs to run now.  This approach is now our recommended way for new comers to the project to install and administer Flyte and there is a [new Helm chart](https://github.com/flyteorg/flyte/tree/master/charts/flyte-binary) also. Documentation has been updated to take this into account. For new installations of Flyte, clusters that do not already have the `flyte-core` or `flyte` charts installed, users can
+```bash
+helm install flyte-server flyteorg/flyte-binary --namespace flyte --values your_values.yaml
+```
+
+#### New local demo environment
+Users may have noticed that the environment provided by `flytectl demo start` has also been updated to use this new style of deployment, and internally now installs this new Helm chart. The demo cluster now also exposes an internal docker registry on port `30000`. That is, with the new demo cluster up, you can tag and push to `localhost:30000/yourimage:tag123` and the image will be accessible to the internal Docker daemon. The web interface is still at `localhost:30080`, Postgres has been moved to `30001` and the Minio API (not web server) has been moved to `30002`.
+
+### Human-in-the-loop Workflows
+Users can now insert sleeps, approval, and input requests, in the form of gate nodes. Check out one of our earlier [issues](https://github.com/flyteorg/flyte/issues/208) for background information.
+
+```python
+from flytekit import wait_for_input, approve, sleep
+
+@workflow
+def mainwf(a: int):
+    x = t1(a=a)
+    s1 = wait_for_input("signal-name", timeout=timedelta(hours=1), expected_type=bool)
+    s2 = wait_for_input("signal name 2", timeout=timedelta(hours=2), expected_type=int)
+    z = t1(a=5)
+    zzz = sleep(timedelta(seconds=10))
+    y = t2(a=s2)
+    q = t2(a=approve(y, "approvalfory", timeout=timedelta(hours=2)))
+    x >> s1
+    s1 >> z
+    z >> zzz
+    ...
+```
+
+These also work inside `@dynamic` tasks. Interacting with signals from flytekit's remote experience looks like
+```python
+from flytekit.remote.remote import FlyteRemote
+from flytekit.configuration import Config
+r = FlyteRemote(
+    Config.auto(config_file="/Users/ytong/.flyte/dev.yaml"),
+   default_project="flytesnacks",
+   default_domain="development",
+)
+r.list_signals("atc526g94gmlg4w65dth")
+r.set_signal("signal-name", "execidabc123", True)
+```
+
+## Looking Ahead
+In the coming release, we are focusing on...
+1. [Out of core plugin](https://unionai.atlassian.net/wiki/spaces/FLYTE/pages/47153191/Out+of+core+plugins+Prove+Value): Make backend plugin scalable and easy to author. No need of code generation, using tools that MLEs and Data Scientists are not accustomed to using.
```

Comment:
@pingsutw is this doc restricted to the union team in confluence? if so we probably shouldn't link to it in the public release notes.

Reply:
updated the link, sorry, I thought that is public